start.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+cd frontend 
+yarn install
+yarn start & # `&` let's the script start in the background
+
+cd ../backend
+npm install
+npm start
+
+# If you get 'Permission denied then make this an executable: `chmod +x script-name-here.sh`
```

Comment:
You can set permissions before committing the file, they will be saved in the repo.